static/admin/config.yml

```diff
@@ -0,0 +1,356 @@
+backend:
+  name: git-gateway
+  repo: ec-europa/ecl-twig
+  branch: develop
+  squash_merges: true
+  commit_messages:
+    create: 'feat({{collection}}): create {{collection}} “{{slug}}”'
```

Comment:
This pattern looks a bit strange looking at the open pull requests.

Reply:
it looks ok, to me.
`feat(Component docs): update Component docs “ecl_compliance”`
where collection is "Component docs" and "slug" is the item name.